src/sanitize-argument.ts

```diff
@@ -55,6 +56,20 @@ export async function sanitize_all(argv: any) {
     throw err;
   });
 
+  // sanitize Custom Main Page
+  if (argv.customMainPage) {
+    argv.customMainPage = argv.customMainPage.replace(/ /g,'_');
+    await axios.get(`${mwUrl}/${mwWikiPath || defaultWikiPath}${argv.customMainPage.replace(/ /g,'_')}`)
+    .then((res) => {
+      if(res.status !== 200) {
+        throw new Error(`customMainPage doesn't return 200 status code`);
+      }
+    })
+    .catch((err) => {
+      throw err;
```

Comment:
Why we have a ctsch() here, considering that we have one at a higher level araiund sanitizeAll()

Reply:
yeah can be removed